go/base/context.go

```diff
@@ -99,6 +99,8 @@ type MigrationContext struct {
 	ConfigFile        string
 	CliUser           string
 	CliPassword       string
+	UseTLS            bool
+	TlsCACertificate  string
```

Comment:
Typical Go style would have this named `TLSCACertificate`
https://github.com/golang/go/wiki/CodeReviewComments#initialisms